src/olympia/lib/settings_base.py

```diff
@@ -1599,7 +1599,7 @@ def get_redis_settings(uri):
 # Time in seconds for how long a JWT auth token created by developers with
 # their API key can live. When developers are creating auth tokens they cannot
 # set the expiration any longer than this.
-MAX_APIKEY_JWT_AUTH_TOKEN_LIFETIME = 60
+MAX_APIKEY_JWT_AUTH_TOKEN_LIFETIME = 5 * 60
```

Comment:
Thanks for doing the math here instead of making me do it in reverse a couple months from now!